### DISCUSSION TITLE
Does Lighthouse clear the browser's cache ,when it runs testing?

### DISCUSSION THREAD

**Discussion by JICJ** [GoogleChrome/lighthouse#11758] [Q&A] [answered]
Does Lighthouse clear the browser's cache ,when it runs testing?

**Comment by patrickhulce** [comment, marked as answer]
Yep! The `--disable-storage-reset` flag and unchecking the "Clear storage" box in DevTools can turn this behavior off.

**Comment by JICJ** [comment]
When I use Lighthouse in Chrome DevTools, how can I config that Lighthouse clear the cache ,when it runs testing?

**Reply by patrickhulce** [reply]
> When I use Lighthouse in Chrome DevTools, how can I config that Lighthouse clear the cache ,when it runs testing?

> unchecking the "Clear storage" box in DevTools can turn this behavior off.


Don't uncheck the "Clear storage" box :) It will clear the cache by default if you haven't done anything.

**Comment by jimmason1001** [comment]
disable lighthouse from running
